README.md

```diff
@@ -12,17 +12,24 @@ _WIP_
 
 This guide will walk you through the steps to configure and run the Azure Key Vault provider for Secret Store CSI driver on Kubernetes.
 
-### Install the Secrets Store CSI Driver 
+### Install the Secrets Store CSI Driver and Azure keyvault provider
 **Prerequisites**
 
 Recommended Kubernetes version: 
 - For linux - v1.16.0+
 - For windows - v1.18.0+
 
+### [Installation using Helm](charts/csi-secrets-store-provider-azure/README.md)
```

Comment:
Instead of a hyperlink and a header, which users can miss, can we change to:

**Deployment using Helm**
Follow this [guide](charts/csi-secrets-store-provider-azure/README.md) to install using Helm.
<details>
<summary><strong>[ALTERNATIVE DEPLOYMENT OPTION] Using Deployment Yamls</strong></summary>
</details>